marketplace/src/lib.rs

```diff
@@ -175,8 +217,8 @@ pub mod pallet {
 
 	#[pallet::error]
 	pub enum Error<T> {
-		/// Account not allowed to list NFTs on that marketplace.
-		AccountNotAllowedToList,
+		/// Not Allowed To List On MP
+		NotAllowedToList,
```

Comment:
All new errors needs to be added to the end of the list. Otherwise you risk breaking existing apps.